src/frontend/sider/Fagsak/Personlinje/Personlinje.tsx

```diff
@@ -15,66 +21,150 @@ import type { IPersonInfo } from '../../../typer/person';
 import { Datoformat, isoStringTilFormatertString } from '../../../utils/dato';
 import { formaterIdent, hentAlder, millisekunderIEttÅr } from '../../../utils/formatter';
 
-interface IProps {
-    bruker?: IPersonInfo;
+interface PersonlinjeProps {
+    søker?: IPersonInfo;
     minimalFagsak?: IMinimalFagsak;
     behandling?: IBehandling;
 }
 
-const Personlinje: React.FC<IProps> = ({ bruker, minimalFagsak, behandling }) => {
-    const { harInnloggetSaksbehandlerSkrivetilgang } = useAppContext();
+interface PersonlinjeIkonProps {
+    fagsakType?: FagsakType;
+    kjønn: string;
+    alder: number;
+}
+
+const PersonlinjeIkon: React.FC<PersonlinjeIkonProps> = ({ fagsakType, kjønn, alder }) => {
```

Comment:
Her kan du også sikkert bare droppe `React.FC<T>`, ser det gjelder andre komponenter her også 

```suggestion
const PersonlinjeIkon = ({ fagsakType, kjønn, alder }: PersonlinjeIkonProps) => {
```